WalletWasabi.Backend/Global.cs

```diff
@@ -56,12 +56,12 @@ public async Task InitializeAsync(Config config, CoordinatorRoundConfig roundCon
 			await InitializeP2pAsync(config.Network, config.GetBitcoinP2pEndPoint(), cancel);
 
 			CoordinatorParameters coordinatorParameters = new(DataDir);
-			HostedServices.Register<WabiSabiCoordinator>(new WabiSabiCoordinator(coordinatorParameters, RpcClient), "WabiSabi Coordinator");
+			using var wabiSabiCoordinator = new WabiSabiCoordinator(coordinatorParameters, RpcClient);
+			HostedServices.Register<WabiSabiCoordinator>(wabiSabiCoordinator, "WabiSabi Coordinator");
 
 			if (roundConfig.FilePath is { })
 			{
-				HostedServices.Register<ConfigWatcher>(
-					new ConfigWatcher(
+				using var configWatcher = new ConfigWatcher(
```

Comment:
This is not correct actually. This is because you want `configWather` to live until the application ends. Your change makes sure that `configWatcher` is disposed on line 81.

As a rule of thumb, `using` should not be used when that object is passed to `HostedServices` because that container diposes components registered to it (see `Startup.cs` file).